extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -47,8 +51,28 @@ oppia.directive('oppiaInteractiveImageClickInput', [
               '/' + encodeURIComponent($scope.filepath)) : null);
           $scope.mouseX = 0;
           $scope.mouseY = 0;
+          $scope.interactionIsActive = !$scope.getLastAnswer();
           $scope.currentlyHoveredRegions = [];
           $scope.allRegions = imageAndRegions.labeledRegions;
+          $scope.updateCurrentlyHoveredRegions = function() {
+            for (var i = 0; i < imageAndRegions.labeledRegions.length; i++) {
+              var labeledRegion = imageAndRegions.labeledRegions[i];
+              var regionArea = labeledRegion.region.area;
+              if (regionArea[0][0] <= $scope.mouseX &&
+                  $scope.mouseX <= regionArea[1][0] &&
+                  regionArea[0][1] <= $scope.mouseY &&
+                  $scope.mouseY <= regionArea[1][1]) {
+                $scope.currentlyHoveredRegions.push(labeledRegion.label);
+              }
+            }
+          };
+          if ($scope.getLastAnswer()) {
+            /*The following lines permanently highlight the learner's previous
```

Comment:
... highlight the learner's last answer for this card.

Reply:
Done